apis/multiarch/v1alpha1/clusterPodPlacementConfig_conversion.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2023 Red Hat, Inc.
+Copyright 2025 Red Hat, Inc.
```

Comment:
```suggestion
Copyright 2023 Red Hat, Inc.
```

just simplifying here for reducing the amount of changes we read while reviewing